src/API/Init.php

```diff
@@ -74,7 +74,7 @@ public function rest_api_init() {
 			'Automattic\WooCommerce\Admin\API\Themes',
 		);
 
-		if ( Loader::is_onboarding_enabled() ) {
+		if ( Loader::is_onboarding_enabled() || Loader::is_feature_enabled( 'shipping-label-banner' ) ) {
```

Comment:
Could we add a comment here that details which endpoint(s) are used by the shipping label banner just so if/when in the future we refactor this a bit, it will be easy to see why this change was made?